### DISCUSSION TITLE
DllPlugin: When processing multiple chunks, does the `path` option must have a string template?

### DISCUSSION THREAD

**Discussion by cwj0417** [webpack/webpack#18200] [Q&A] [answered]
When generating Dll files, It seems like every chunk should emit a manifest file.

If we not config `[name]` (or `[hash]`) placeholder in `path` option, all chunks was written in a single file and leads to undesirable result.

If what I described above is correct, I considered we could add some explanations to document and print warning on console. and can I send a PR improve this?

To reproduce: change `path` option in `examples/dll/webpack.config.js` to `path.join(__dirname, "dist", "manifest.json")`, and run `node build.js` for **several times**, manifest.json is occasionally written to an illegal json.

(I don' t think this is a bug, so I raise this issue with "other" template)

**Comment by alexander-akait** [comment, marked as answer]
> To reproduce: change path option in examples/dll/webpack.config.js to path.join(__dirname, "dist", "manifest.json"), and run node build.js for several times, manifest.json is occasionally written to an illegal json.

Yes, we can improve and output an error, feel free to send a PR to improve it
